packages/login/src/components/PasswordRequest/PasswordRequest.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {FormattedMessage, intlShape} from 'react-intl'
+import {Pages} from '../../types/Pages'
+import * as Tocco from 'tocco-ui'
+
+import '../Login/styles.scss'
+
+export class PasswordRequest extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      username: ''
+    }
+  }
+
+  handleSubmit(e) {
+    e.preventDefault()
+    this.props.requestPassword(this.state.username)
+  }
+
+  handleUsernameChange(e) {
+    this.setState({
+      username: e.target.value
+    })
+  }
+
+  render() {
+    return (
+      <div className="login-form">
+        {this.props.showTitle && <h1><FormattedMessage id="client.login.passwordRequest.title"/></h1>}
+        <p><FormattedMessage id="client.login.passwordRequest.introduction"/></p>
+        <form>
```

Comment:
I think we should register the `handleSubmit` event handler on the form (not as click handler on the button). And the submit button should be of type `submit` (without click handler).